tests/policies/test_exploration_wrapper.py

```diff
@@ -57,11 +57,46 @@ def test_switch():
     assert num_constant > 450
 
 
+def test_switch_prob():
+    # Ensure that we test both the random and the wrapped policy
```

Comment:
Meaning of comment unclear. Code block immediately following it checks that the policy is never switched (when `switch_prob=0.0`) which does not seem to connect to this.